Cake.Recipe/Content/build.cake

```diff
@@ -132,10 +127,7 @@ BuildParameters.Tasks.DotNetCoreRestoreTask = Task("DotNetCore-Restore")
 {
     DotNetCoreRestore(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreRestoreSettings
     {
-        Sources = new [] {
-            "https://api.nuget.org/v3/index.json",
-            "https://www.myget.org/F/cake-contrib/api/v3/index.json"
-        },
+        Sources = BuildParameters.NuGetSources,
```

Comment:
I could be wrong, but I believe this could cause problems.
AFAIK the .NET Core utility isn't compatible with nuget v2 sources, only with v3.